dev/devicelab/bin/tasks/build_aar_plugin_test.dart

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:path/path.dart' as path;
+
+final String gradlew = Platform.isWindows ? 'gradlew.bat' : 'gradlew';
+final String gradlewExecutable = Platform.isWindows ? gradlew : './$gradlew';
+
+/// Tests that AARs can be built on plugin projects.
+Future<void> main() async {
+  await task(() async {
+
+    section('Find Java');
+
+    final String javaHome = await findJavaHome();
+    if (javaHome == null)
+      return TaskResult.failure('Could not find Java');
+    print('\nUsing JAVA_HOME=$javaHome');
+
+    section('Create plugin project');
+
+    final Directory tempDir = Directory.systemTemp.createTempSync('flutter_module_test.');
+    final Directory projectDir = Directory(path.join(tempDir.path, 'hello'));
+    try {
+      await inDirectory(tempDir, () async {
+        await flutter(
+          'create',
+          options: <String>[
+            '--org', 'io.flutter.devicelab',
+            '--template', 'plugin',
+            'hello',
+          ],
+        );
+      });
+
+      section('Build release AAR');
+
+      await inDirectory(projectDir, () async {
+        await flutter(
+          'build',
+          options: <String>['aar', '--verbose'],
+        );
+      });
+
+      final String repoPath = path.join(
+        projectDir.path,
+        'build',
+        'outputs',
+        'repo',
+      );
+
+      final File releaseAar = File(path.join(
+        repoPath,
+        'io',
+        'flutter',
+        'devicelab',
+        'hello',
+        'hello_release',
+        '1.0',
+        'hello_release-1.0.aar',
+      ));
```

Comment:
Can we check that both 32 and 64 are in there?

Reply:
plugins don't have such.

Reply:
sorry, this comment should have been for the build_aar_module_test